src/containers/ProfilePage/index.jsx

```diff
@@ -0,0 +1,91 @@
+import React from "react";
+import { Container, Row } from "react-bootstrap";
+import UserDetail from "../../components/UserDetail";
+import UserEmail from "../../components/UserEmail";
+import UserPassword from "../../components/UserPassword";
+import defaultProfileImage from "../../images/default-profile-image.png";
+import "./index.scss";
+
+export default function ProfilePage() {
+  const [elementSelected, setElementSelected] = React.useState("userDetail");
+
+  return (
+    <Container fluid="md" className="profileContainer">
+      <Row className="profilePortrait">
+        <h2>Profile</h2>
+        <div className="userPortrait">
+          <img src={defaultProfileImage} alt="User portrait" />
+        </div>
+      </Row>
+      <Row className="section userDetail">
+        <button
+          className="sectionTitle"
+          onClick={() => setElementSelected("userDetail")}
+          type="button"
+        >
+          <i
+            className={
+              elementSelected === "userDetail" ? "arrow down" : "arrow right"
```

Comment:
where does the arrow down and right come from?
is it  built in react or an installed package? I don't see it imported.
I am just asking to personally understand.

Reply:
or did you add the arrow in CSS? since it's in className

Reply:
they are classes, you can find them in the CSS style file